src/gdrive/README.md

```diff
@@ -31,8 +31,9 @@ The server provides access to Google Drive files:
 3. [Configure an OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent) ("internal" is fine for testing)
 4. Add OAuth scope `https://www.googleapis.com/auth/drive.readonly`
 5. [Create an OAuth Client ID](https://console.cloud.google.com/apis/credentials/oauthclient) for application type "Desktop App"
-6. Download the JSON file of your client's OAuth keys
-7. Rename the key file to `gcp-oauth.keys.json` and place into the root of this repo (i.e. `servers/gcp-oauth.keys.json`)
+6. Add http://localhost:3000/oauth2callback as a redirect URI
```

Comment:
@dsp-ant Hope you don't mind, but I found this extra step was important to get the mcp server to work regardless of Docker/npx